### PR TITLE
Lint cleanup

### PR DESCRIPTION
**Pull request by swansontec**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203287918505544

### CODE REVIEW

File: src/rate/bitmax.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export function makeBitMaxPlugin(opts: EdgeCorePluginOptions): EdgeRatePlugin {
             }
           ]
         } catch (e) {
-          log.warn(`Issue with Bitmax rate data structure ${e}`)
+          log.warn(`Issue with Bitmax rate data structure ${String(e)}`)
```
Comment (peachbits) [inline src/rate/bitmax.ts:38]: We don't use the rate plugins anymore so we could delete them instead

File: src/rate/currencyconverterapi.ts [hunk 2]
```diff
@@ -5,6 +5,10 @@ import {
   EdgeRatePlugin
 } from 'edge-core-js/types'
 
+const asInitOptions = asObject({
```
Comment (peachbits) [inline src/rate/currencyconverterapi.ts:8]: Missed spookyswap and tombswap. Those plugins are buried real deep in the folder hierarchy for some reason.

**Review by peachbits — CHANGES_REQUESTED** [review]

File: src/rate/currencyconverterapi.ts (hunk 2 above)
Comment (swansontec) [reply src/rate/currencyconverterapi.ts:8]: Added them.

File: src/rate/bitmax.ts (hunk 1 above)
Comment (swansontec) [reply src/rate/bitmax.ts:38]: I agree, but I think we need to do this in another PR.

**Review by peachbits — DISMISSED** [review]

**Review by peachbits — APPROVED** [review]

**Pull request merged**